stories/host-dashboard/UnhostAccountModal.stories.mdx

```diff
@@ -20,7 +23,11 @@ export const DefaultStory = props => <UnhostAccountModal {...props} />;
       },
     }}
   >
-    <UnhostAccountModal collective={webpackCollective} host={webpackCollective.host} />
+    <UserContext.Provider value={{ LoggedInUser: loggedInUser, loadingLoggedInUser: false }}>
+      <ToastProvider>
+        <UnhostAccountModal collective={webpackCollective} host={webpackCollective.host} />
+      </ToastProvider>
+    </UserContext.Provider>
```

Comment:
Ok for the `UserContextProvider` as it's difficult to figure a right default, but the `ToastProvider` should go in the [default decorators](https://github.com/opencollective/opencollective-frontend/blob/07f5fe7700d865f54f175fe6afccaf683d259631/.storybook/preview.js#L1) no?

Reply:
Oh, that makes sense. I've made the change. 👍🏽